src/test/java/test/Test1.java

```diff
@@ -36,7 +38,8 @@ public void includedGroups() {
 
         tng.run();
 
-        assertThat(listener.getSucceedMethodNames()).containsExactly("method1", "method3");
+        List<String> expected = Arrays.asList("method1", "method3");
+        doAssert(listener.getSucceedMethodNames(), expected);
```

Comment:
> the order is all governed by how Java's reflection yields those methods

Does it mean it could change with different java version? Or it is the order in the file from top to bottom?
The problem is the modification of the current behavior.

> Where would that new option reside?
Just an idea: It could be something like "alphaOrder" where methods of a class will be sorted by their name and linked together.
The goal is to have a way to keep the current behavior (or have a way to restore it).